Project2015To2017.Core/Writing/ProjectWriteOptions.cs

```diff
@@ -0,0 +1,24 @@
+using System;
+using System.IO;
+
+namespace Project2015To2017.Writing
+{
+	public class ProjectWriteOptions
+	{
+		/// <summary>
+		/// Make backups of changed and deleted files before changing/deleting them
```

Comment:
A bit awkward.

Reply:
I'm definitely open to alternatives

Reply:
Proposed:
`Make backup copies of select project files before their modification or deletion.`

Reply:
good idea, I'll change it to something similar (just dropping the 'select' as I'm not sure what it means in this context, and saying files as people might think 'project files' means just .csproj files):

`Make backup copies of files before their modification or deletion`